src/domain/challenge/challenge/challenge.entity.ts

```diff
@@ -42,7 +42,7 @@ export class Challenge extends BaseChallenge implements IChallenge {
   parentHub?: Hub;
 
   @Column()
-  hubID!: string;
+  hubID?: string;
```

Comment:
Currently, `hubID` is mandatory. We don't really support nested `Challenge`. Why was this changed?

Reply:
There has been a change in VS Code which means that it no longer considers the repositories between Hub / Challenge / Opportunity the same as BaseChallenge if there are mandatory fields in the entity definition.